modules/data-streams/src/internalClusterTest/java/org/elasticsearch/datastreams/TSDBIndexingIT.java

```diff
@@ -167,39 +165,6 @@ public void testTimeRanges() throws Exception {
         }
     }
 
-    public void testTsdbTemplatesWithComponentTemplate() throws Exception {
```

Comment:
This test doesn't make a lot of sense now that `index.mode` setting is no longer automatically added to backing indices.

Reply:
In exchange of removing this, I added some extra test at the end of this file that test validation that should kick in when creating template that creates time series data streams but index.mode or index.routing_path is missing.